### DISCUSSION TITLE
where is the nickname from github saved?

### DISCUSSION THREAD

**Discussion by melonges** [zbirenbaum/copilot.lua#120] [Q&A] [answered]
I want to log in through another account, but even after reinstalling the plugin, the nickname is saved somewhere and logged in using it
how is my nickname from github determined?
<img width="528" alt="image" src="https://user-images.githubusercontent.com/70666395/217898610-ac5ebc5f-bc79-4c44-a56d-507c2a522f74.png">

**Comment by MunifTanjim** [comment, marked as answer]
> I want to log in through another account

You can run `:Copilot auth signout` to sign out of current account.

> even after reinstalling the plugin, the nickname is saved somewhere and logged in using it
how is my nickname from github determined

https://github.com/zbirenbaum/copilot.lua/issues/101#issuecomment-1374404716

The auth information is usually stored in `~/.config/github-copilot/hosts.json` file.
